imports/plugins/custom/landingPage/client/templates/footer.html

```diff
@@ -32,4 +48,25 @@ <h5 className="lead">Contact Us</h5>
       </div>
     </div>
   </div>
-</template>
\ No newline at end of file
+
+  <!-- <script>window.twttr = (function(d, s, id) {
+    var js, fjs = d.getElementsByTagName(s)[0],
+      t = window.twttr || {};
+    if (d.getElementById(id)) return t;
+    js = d.createElement(s);
+    js.id = id;
+    js.src = "https://platform.twitter.com/widgets.js";
+    fjs.parentNode.insertBefore(js, fjs);
+  
+    t._e = [];
+    t.ready = function(f) {
+      t._e.push(f);
+    };
+  
+    return t;
+  }(document, "script", "twitter-wjs"));</script> -->
+</template>
+<!-- <script dangerouslySetInnerHTML={{ __html: `window.twttr=( function(d, s, id) { var js, fjs=d .getElementsByTagName(s)[0],
+  t=w indow.twttr || {}; if (d.getElementById(id)) return t; js=d .createElement(s); js.id=i d; js.src="https://platform.twitter.com/widgets.js"
+  ; fjs.parentNode.insertBefore(js, fjs); t._e=[ ]; t.ready=f unction(f) { t._e.push(f); }; return t; }(document, "script",
+  "twitter-wjs"));`, }} /> -->
```

Comment:
Add a new line after line `72`

Reply:
remove commented codes @ddouglasz

Reply:
On it guys. Thanks for the extra eyes 👀👀